modules/internal/BytesStringCommon.chpl

```diff
@@ -673,6 +679,7 @@ module BytesStringCommon {
                    dst_off=lhs.buffLen);
       lhs.buffLen = newLength;
       lhs.buff[newLength] = 0;
+      if t == string then incrementCodepoints(lhs, rhs);
```

Comment:
didn't we change this to setCodepoints or something? One alternative can be `safeAddNumCodepoints`

Reply:
I think that the original name was setCodepoints and we decided to change it to incrementCodepoints. Do you prefer the name safeAddNumCodepoints?

Reply:
Yikes.. I see that that's what I proposed above. And now that I looked at what it does again, I feel better about `incrementCodepoints` Sorry about the noise